.rubocop.yml

```diff
@@ -13,23 +13,23 @@ AllCops:
 Gemspec/RequireMFA:
   Enabled: false
 
-Layout/ArgumentAlignment:
-  Enabled: false
+Style/TrailingCommaInHashLiteral:
+  Enabled: True
+  EnforcedStyleForMultiline: consistent_comma
+
+Style/TrailingCommaInArrayLiteral:
+  Enabled: True
+  EnforcedStyleForMultiline: consistent_comma
+
+Style/TrailingCommaInArguments:
+  Enabled: True
+  EnforcedStyleForMultiline: comma
 
 Layout/LineLength:
   Exclude:
     - acceptance/**/*.rb
     - spec/**/*.rb
 
-Layout:
-  Enabled: false
-
-Metrics:
-  Enabled: false
```

Comment:
Can we keep Metrics disabled until a later PR? I think this is now just moving things to the TODO file and not fixing anything.

Reply:
done

Reply:
I also disabled Style and Layout for now, to keep the file smaller.